versioned_docs/version-v5/native/background-fetch.md

```diff
@@ -57,16 +56,17 @@ For more detail, please see https://github.com/transistorsoft/cordova-plugin-bac
   ]}
 >
   <TabItem value="Capacitor">
-    <CodeBlock className="language-shell">
-      $ npm install cordova-plugin-background-fetch {'\n'}$ npm install @awesome-cordova-plugins/background-fetch {'\n'}
-      $ ionic cap sync
-    </CodeBlock>
+
+    $ npm install cordova-plugin-background-fetch
+    $ npm install @awesome-cordova-plugins/background-fetch
+    $ ionic cap sync
+
   </TabItem>
   <TabItem value="Cordova">
-    <CodeBlock className="language-shell">
-      $ ionic cordova plugin add cordova-plugin-background-fetch {'\n'}$ npm install
-      @awesome-cordova-plugins/background-fetch {'\n'}
-    </CodeBlock>
+
```

Comment:
Yes, the change is intentional here. 

The project currently has a weird formatting issue, where every-other-time you run prettier, it will either format these files with additional whitespace or remove the additional whitespace. 

To avoid this scenario and have consistent prettier formatting across runs, I removed the `<CodeBlock>` element from the offending pages and used the `$` syntax highlighting directly for the code blocks. It does remove _some_ syntax highlighting, but I think that is acceptable for a few reasons:
1. The native plugin pages no longer exist (these are legacy pages)
2. Prettier formatting inconsistently will block our CI process once the `git diff` check is enabled.
3. The code blocks are still formatted/styled, they just don't have syntax highlighting on parts of the bash command (which I don't think exists when using markdown formatting anyways).